src/SourceBuild/tarball/content/ci.yml

```diff
@@ -0,0 +1,19 @@
+variables:
+  NugetSecurityAnalysisWarningLevel: none
```

Comment:
Is this variable used?  From your feedback, it doesn't seem to.  I think it is worth reaching out to OpenSourceEngSupport@microsoft.com on why this doesn't work.

Reply:
There are some places setting this in dotnet - https://github.com/dotnet/project-system/blob/5930b5b0dc5afb547eb399cd71445d43338647cb/eng/pipelines/pull-request.yml#L46

Reply:
Perhaps the problem is not multiple feeds rather there is no clear first?

Reply:
It seems to me that this variable does nothing and might have been a placeholder while teams fix their NuGet configs. I have sent an email to that alias and CC'ed you, and I will remove the variable from the pipeline as well.